backend/app/tests/conftest.py

```
@@ -56,7 +56,10 @@ def db() -> Generator[Session, None, None]:
     # We're using the engine that was set up in setup_test_environmen
     from app.core.db import engine
 
-    # 密码已经在setup_test_environment中设置，这里直接初始化数据库
+    # 确保测试用的超级用户密码为 "telepace"，满足至少8个字符的要求
+    original_password = settings.FIRST_SUPERUSER_PASSWORD
```

Ruff (0.11.9)

Local variable `original_password` is assigned to but never used

Remove assignment to unused variable `original_password`

(F841)